File: DealApp.API/Models/Renter.cs
using System;

namespace DealApp.API.Models
{
    public class Renter
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public Int64 MobileNo1 { get; set; }
        public Int64 MobileNo2 { get; set; }
        public bool? IsActive { get; set; }
        public bool? IsDeleted { get; set; }
        public string CreatedBy { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string ModifyBy { get; set; }
        public DateTime? ModifyDate { get; set; }
    }
}

File: DealApp-SPA/src/app/_models/renter.ts
export interface Renter {
    id: number;
    name: string;
    mobileNo1: number;
    mobileNo2: number;
    isActive?: boolean;
    isDeleted?: boolean;
    createdBy?: string;
    createdDate?: Date;
    modifiedBy?: string;
    modifiedDate?: Date;
}
